.github/workflows/on_PR_windows_matrix.yml

```diff
@@ -115,6 +115,7 @@ jobs:
           update: true
           install: >-
             base-devel
+            git
```

Comment:
No need to build, it looks like `libinih` is available for MSYS2 (but indeed not for Cygwin)

Reply:
It's been a couple of weeks since I got MSYS2 working, but I think the problem is that I need a "libinih-devel" library. The "libinih" library doesn't include everything that you need to build exiv2.

I've been battling with this PR for a few weeks in my own fork: https://github.com/kevinbackhouse/exiv2/pull/5. Yesterday I created this new PR because I wanted to squash the history, but accidentally created it on the main exiv2 repo. So I deleted this PR and created a new one here: https://github.com/kevinbackhouse/exiv2/pull/6. I have now finally figured out the solution to the cygwin build failures, so I'll polish it up and submit a new PR to exiv2 later today.

Reply:
> The "libinih" library doesn't include everything that you need to build exiv2.

There is no such -devel library on MSYS2 (or Arch, from which it derives). If there's stuff missing, let's report it as a packaging issue upstream, if you really want to pursue this.

Reply:
I remember now: MSYS2 does have a devel package: https://packages.msys2.org/package/libinih-devel. But I couldn't get it to work. I think we need a mingw-w64-ucrt-x86_64 version, like this zlib one: https://packages.msys2.org/base/mingw-w64-zlib

Reply:
Ah, these are MSYS packages (for the MSYS environment, which is like Cygwin), not MinGW (native Windows) packages as I too quickly judged... So we'd need to request/create them if you like?